terragrunt/org_account/spend_notifier/lambdas/spend_notifier/spend_notifier.js

```diff
@@ -13,18 +13,33 @@ exports.handler = async (event) => {
 
   const accountCost = await getAccountCost()
   let accounts = await getAccounts()
+
+  // get all scratch accounts that exceeded the threshold of $500 yesterday
+  const scratchAccountsExceedingThreshold = await getScratchAccountsExceedingThreshold()
+
   let accountIncreases = {}
+  let scratchAccountsAffected = {}
+
   Object.keys(accounts).forEach(key => {
     if(accountCost.hasOwnProperty(key)){
       accounts[key]["Cost"] = accountCost[key]
+      // determine if the account is a scratch account
+      accounts[key]["Name"].toLowerCase().includes("scratch") ? accounts[key]["isScratch"] = true : accounts[key]["isScratch"] = false 
     } else {
       accounts[key]["Cost"] = 0
     }
-    // if there is a 35% increase in costs for yesterady vs day before, add to accountIncreases
-    if(dailyAccountCost.hasOwnProperty(key) && dailyAccountCost[key] > 35) {
+
+    // if the account is not a scratch account and there is a 35% increase in costs for yesterday vs day before, add to accountIncreases
+    if(dailyAccountCost.hasOwnProperty(key) && dailyAccountCost[key] > 35 && !accounts[key]["Name"].toLowerCase().includes("scratch")) {
```

Comment:
So this is a little bit weird here and is confusing. What we want is all accounts that have shown a 35% increase in cost but are not scratch accounts since scratch accounts will always show such an increase if people are using them. For scratch accounts, we want to report if they go over $500. That is why in the if statement above, we want to report all accounts that have seen at least 35% spike in cost and are not scratch accounts. Does it make sense? Sorry if it is confusing and unclear.